### PR TITLE
feat: add instances backup help

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

f3528150 — feat: add instances backup help

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/instances/help/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "pci_projects_project_instances_help_backup_title": "Pour créer un backup :",
+  "pci_projects_project_instances_help_backup_description": "Cliquez sur le bouton action de l'instance souhaitée, puis cliquez sur Créer un backup.",
+  "pci_projects_project_instances_help_dismiss": "Ne plus me montrer ce message"
+}
+
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/instances/help/translations/Messages_fr_FR.json]: Extra empty lines.

File: packages/manager/modules/pci/src/projects/project/instances/help/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "pci_projects_project_instances_help_backup_title": "Pour créer un backup :",
+  "pci_projects_project_instances_help_backup_description": "Cliquez sur le bouton action de l'instance souhaitée, puis cliquez sur Créer un backup.",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/instances/help/translations/Messages_fr_FR.json]:
```suggestion
  "pci_projects_project_instances_help_backup_description": "Cliquez sur le bouton action de l'instance souhaitée, puis cliquez sur « Créer un backup ».",
```

File: packages/manager/modules/pci/src/projects/project/instances/instances.controller.js [hunk 3]
```diff
@@ -53,4 +58,43 @@ export default class CloudProjectComputeInfrastructureListCtrl {
...
+    if (type === this.displayHelp) {
+      this.displayHelp = null;
+    }
+  }
+
+  hideHelp(type) {
+    if (type === this.displayHelp) {
+      this.updateHelp = true;
+
+      this.ovhUserPref.assign(this.getHelpPreferenceKey(), false)
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/instances/instances.controller.js]: Couldn't be returned?

File: packages/manager/modules/pci/src/projects/project/instances/help/backup.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<p>
+    <span class="d-block" data-translate="pci_projects_project_instances_help_backup_title"></span>
+    <span data-translate="pci_projects_project_instances_help_backup_description"></span>
+</p>
+
+<button type="button" class="oui-link"
+    data-ng-click="$ctrl.hideHelp('backup')"
+    data-translate="pci_projects_project_instances_help_dismiss"
+    data-ng-show="!$ctrl.updateHelp"></button>
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/instances/help/backup.html]:
```suggestion
    data-ng-if="!$ctrl.updateHelp"></button>
```
?

File: packages/manager/modules/pci/src/projects/project/instances/help/backup.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+<oui-spinner data-size="s"
+    data-ng-show="$ctrl.updateHelp"></oui-spinner>
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/instances/help/backup.html]:
```suggestion
    data-ng-if="$ctrl.updateHelp"></oui-spinner>
```
?

File: packages/manager/modules/pci/src/projects/project/instances/instances.controller.js [hunk 6]
```diff
@@ -53,4 +58,43 @@ export default class CloudProjectComputeInfrastructureListCtrl {
       instance,
     );
   }
+
+  checkHelpDisplay() {
+    return new Promise((resolve) => {
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/instances/instances.controller.js]:
```suggestion
    return this.help
      ? this.ovhUserPref.getValue(this.getHelpPreferenceKey())
          .then((value) => {
            this.displayHelp = value && this.help;
          })
          .catch(() => {
            this.displayHelp = this.help;
          });
        : Promise.resolve()
```

**Pull request merged**